about.html

```diff
@@ -0,0 +1,160 @@
+    <title>AlterTime: Mage Class Hall</title>
+
+    <!-- fontawesome Cdn Link 6.2.1 -->
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/css/all.min.css">
+
+    <!-- Custom css file link -->
+    <link rel="stylesheet" href="css/about.css">
+
+    <!-- Custom JS file link -->
+    <script defer src="js/main.js"></script>
+
+</head>
+
+<body>
+    <!-- Navbar Section Starts here -->
+    <section id="navbar">
+        <nav class="sub__navbar">
+            <div class="socialIcon">
+                <a href="#"><i class="fa-brands fa-facebook-f"></i></a>
+                <a href="#"><i class="fa-brands fa-twitter"></i></a>
+            </div>
+            <div class="sub__navMenu">
+                <ul class="navLink">
+                    <li><a href="#">English</a></li>
+                    <li><a href="#">My Page</a></li>
+                    <li><a href="#">Logout</a></li>
+                </ul>
+            </div>
+        </nav>
+        <nav class="main__navbar">
+            <a href="index.html" class="logo"><img width="60" src="img/mage icon.png" alt="logo"></a>
+            <div class="main__navMenu">
+                <ul class="navLink">
+                    <li><a href="#">About</a></li>
+                    <li><a href="#Program">Program</a></li>
+                    <li><a href="#Join">Join</a></li>
+                    <li><a href="#partner">Sponsor</a></li>
+                    <li><a href="#News">News</a></li>
+                    <li><button type="button" class="main__navBtn">AT campaign</button></li>
+                </ul>
+            </div>
+        </nav>
+    </section>
+    <!-- Navbar Section Ends here -->
+
+    <!-- hamburger -->
+    <div class="hamburger">
+        <span class="bar"></span>
+        <span class="bar"></span>
+        <span class="bar"></span>
+    </div>
+    <!-- hamburger -->
+
+    <!-- about Header Section starts -->
+    <section id="about">
+        <header class="headerBanner">
+            <div class="banner">
+                <h3 class="subHeader">"Hello, Fellow Mages"</h3>
+                <h1 class="mainHeader">World Of Warcraft: Altered Time Mages' Class Hall Event</h1>
+                <div class="underline"></div>
+                <p class="headerPara__mobile">
+                    Lorem, ipsum dolor sit amet consectetur adipisicing elit. Quibusdam veniam consequatur corporis
+                    facere vel reiciendis maxime non sed fugit quos quisquam enim sapiente unde dolores, temporibus
+                    repudiandae molestiae exercitationem officia repellat officiis!
+                </p>
+                <p class="headerPara__desktop">
+                    Lorem, ipsum dolor sit amet consectetur adipisicing elit. Quibusdam veniam consequatur corporis
+                    facere vel reiciendis maxime non sed fugit quos quisquam enim sapiente unde dolores, temporibus
```

Comment:
- [ ] You did a great job working on this section! 💯 👏🏼 However, there's a visual bug when I try to test the mobile version using an iPhone SE dimensions as shown below:

![image](https://user-images.githubusercontent.com/77069011/204029958-1f5a6668-71b4-4e8b-891c-c3a0f22cf2d9.png)

I recommend you make sure your styles work at least until `375px` wide devices, the reason behind this number is that there are a lot of people that have mobile devices with this dimensions. Please consider looking into this and fixing it! You're doing a really good job over here. 🤓 🛠️